c10/util/complex_math.h

```diff
@@ -3,10 +3,11 @@
 #endif
 
 
-namespace std {
-
-// Exponential functions
+#ifndef __HIPCC__
```

Comment:
LOL

Reply:
The problem you are having will be solved by https://github.com/pytorch/pytorch/pull/39829, and this will conflict with https://github.com/pytorch/pytorch/pull/39829. But I don't think this should be blocked by https://github.com/pytorch/pytorch/pull/39829.